src/components/Catalogue.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "../styles/catalogue.scss";
+import { Trans } from "react-i18next";
+
+export default function Catalogue({ children }) {
+  return (
+    <div className="catalogue-box">
+      <h1 className="catalogue-tittle">
```

Comment:
~~tittle~~  ---> title

Reply:
Dunno why but i always write that word like that, silly error.